legacyCore/src/main/res/layout-sw300dp/input_text.xml

```diff
@@ -0,0 +1,28 @@
+<com.google.android.material.textfield.TextInputLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/filledTextField"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/buttonSmall2"
+    android:layout_marginStart="@dimen/paddingLarge3"
+    android:layout_marginTop="@dimen/paddingSmall1"
+    android:layout_marginEnd="@dimen/paddingLarge3"
+    android:hint="@string/create_account_first_name_prompt"
+    app:boxStrokeColor="?legacyOnBackground"
+    app:boxStrokeWidth="0dp"
+    app:boxStrokeWidthFocused="@dimen/strokeSmall2"
+    app:hintTextColor="?legacyOnBackground"
+    app:placeholderTextColor="?legacyOnBackground">
+
+    <!--this is the actual edit text which takes the input-->
+    <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/edit_text"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/input_field_shape"
+        android:maxLines="1"
+        android:text=""
+        android:textColor="?legacyOnBackground"
+        android:textCursorDrawable="@null"
```

Comment:
Why?

Reply:
For some reason during development the cursor was purple, however when testing again on removal it is a normal black. None the less, the issue is that useing styleing and cursorDrawables doesn't allow me to change the appearance of the cursor to match the thicker cursor in compose. I'm not really sure what to do about it.

Reply:
for now the line is entirely removed.

Reply:
annnnnd now its purple again